src/coreclr/tests/src/Common/Directory.Build.targets

```diff
@@ -21,17 +21,34 @@
       <RunTimeDependencyExcludeFiles Include="@(RunTimeDependencyExclude -> '%(FileName)%(Extension)')" />
       <RunTimeDependencyExcludeFiles Include="@(RunTimeDependencyExclude -> '%(FileName).ni%(Extension)')" />
       <RunTimeDependencyExcludeFiles Include="@(RunTimeDependencyExclude -> '%(FileName).pdb')" />
+
       <AllResolvedRuntimeDependencies Include="@(RuntimeCopyLocalItems -> '%(FileName)%(Extension)');@(NativeCopyLocalItems -> '%(FileName)%(Extension)')">
         <File>%(Identity)</File>
       </AllResolvedRuntimeDependencies>
+      
       <RunTimeDependencyCopyLocalFile Include="@(AllResolvedRuntimeDependencies)"  Exclude="@(RunTimeDependencyExcludeFiles)"/>
       <RunTimeDependencyCopyLocal Include="@(RunTimeDependencyCopyLocalFile -> '%(File)')"  />
       <RunTimeDependencyCopyLocal Include="$(TargetingPackPath)/*" />
+      
+      <RunTimeArtifactsExcludeFiles Include="protononjit.dll" />
```

Comment:
That change affects @dotnet/jit-contrib team workflow because now we don't have these altjits in CoreRoot after running `src\coreclr\build-test.cmd`. 
I remember there was an environment variable, like `ci` or similar to distinguish runs in ci from local runs, could that be used to make optimizations in this PR only when it is set and do not affect others?

Reply:
Sure, I was curious what would break :-). Will fix. Can you point me to some pipeline or script I can run to verify the change? Other than that, if this is just about the protononjit / linuxonjit binaries, these aren't very long so I guess we can put them back unconditionally.

Reply:
Check that these altjit.dlls are in the Core_Root. I am not sure if any additional verification is needed, but you can build an individual test and try to run it:
```
set ARCH=x86
set MODE=Checked
set RUNTIME_PATH=F:\git\runtime
%RUNTIME_PATH%\src\coreclr\build-test.cmd skipnative skipmanaged %ARCH% %MODE% /p:LibrariesConfiguration=Release

REM build a test
%RUNTIME_PATH%\.dotnet\dotnet.exe msbuild /p:TargetArchitecture=%ARCH% /p:Configuration=%MODE% /p:LibrariesConfiguration=Release %RUNTIME_PATH%\src\coreclr\tests\src\JIT\Regression\JitBlue\Runtime_33884\Runtime_33884.csproj

set COMPlus_AltJit=*
set COMPlus_AltJitName=protononjit.dll
set CORE_ROOT=%RUNTIME_PATH%\artifacts\tests\coreclr\Windows_NT.%ARCH%.%MODE%\Tests\CORE_ROOT

REM run the test with altjit
%RUNTIME_PATH%\artifacts\tests\coreclr\Windows_NT.x86.Checked\JIT\Regression\JitBlue\Runtime_33884\Runtime_33884\Runtime_33884.cmd
```

Reply:
OK, great. Just tried this (on x64 that I have a complete build for readily available) and I do see some difference - the test you mentioned passes with the normal JIT, with the protononjit it crashes as follows:

<pre>
BEGIN EXECUTION
 "d:\git\runtime2\artifacts\tests\coreclr\Windows_NT.x64.Checked\tests\Core_Root\corerun.exe" Runtime_33884.dll

Assert failure(PID 12860 [0x0000323c], Thread: 14480 [0x3890]): Assertion failed 'tree->IsLocal() || (tree->OperGet() == GT_RET_EXPR) || (tree->OperGet() == GT_CALL) || ((tree->gtOper == GT_ADDR) && varTypeIsSIMD(tree->gtGetOp1()))' in 'System.SpanHelpers:IndexOf(byref,ushort,int):int' during 'Importation' (IL size 899)

    File: D:\git\runtime2\src\coreclr\src\jit\simd.cpp Line: 1197
    Image: d:\git\runtime2\artifacts\tests\coreclr\Windows_NT.x64.Checked\tests\Core_Root\CoreRun.exe

Expected: 100
Actual: -1073740286
END EXECUTION - FAILED
FAILED
</pre>

Is that the expected behavior?

Reply:
That's the expected behavior because the SIMD size is different. It should work if you `set COMPlus_SIMD16ByteOnly=1`